doc/containers.md

```diff
@@ -531,6 +531,10 @@ Disk entries are essentially mountpoints inside the container. They can
 either be a bind-mount of an existing file or directory on the host, or
 if the source is a block device, a regular mount.
 
+LXD supports the following additional source types:
+- [Ceph-rbd](#sourcetype-ceph-rbd): Mount from existing ceph RBD device that is externally managed
+- [Ceph-fs](#sourcetype-ceph-fs): Mount from existing ceph FS device that is externally managed
```

Comment:
This now links to sections that don't exist. So the link should be removed and a few more details added to at least show the expected `source=` syntax for each of the two options.